contracts/anchor/StateRootProvider.sol

```diff
@@ -48,7 +48,7 @@ contract StateRootProvider is CircularBufferUint {
     function getStateRoot(
         uint256 _blockNumber
     )
-        external
+        public
```

Comment:
why public?

Reply:
Reverted this change. It was made public because I used these functions to check if the setup was called in the inherited contracts. I later removed the code but forgot to change this back to external.